utils/results/conversion/sarifparser/sarifparser.go

```diff
@@ -401,16 +531,30 @@ func getDirectDependenciesFormatted(directDependencies []formats.ComponentRow) (
 	return strings.TrimSuffix(formattedDirectDependencies.String(), "<br/>"), nil
 }
 
-func getScaVulnerabilitySarifHeadline(depName, version, issueId string) string {
-	return fmt.Sprintf("[%s] %s %s", issueId, depName, version)
+func getScaVulnerabilitySarifHeadline(depName, version, issueId, watch string) string {
+	headline := fmt.Sprintf("[%s] %s %s", issueId, depName, version)
+	if watch != "" {
+		headline = fmt.Sprintf("%s (%s)", headline, watch)
+	}
+	return headline
 }
 
-func getScaSecurityViolationSarifHeadline(depName, version, key string) string {
-	return fmt.Sprintf("Security violation %s", getScaVulnerabilitySarifHeadline(depName, version, key))
+func getScaSecurityViolationSarifHeadline(depName, version, key, watch string) string {
+	headline := getScaVulnerabilitySarifHeadline(depName, version, key, watch)
+	if watch == "" {
+		return fmt.Sprintf("Security Violation %s", headline)
+	}
+	return headline
 }
 
-func getXrayLicenseSarifHeadline(depName, version, key string) string {
-	return fmt.Sprintf("License violation [%s] in %s %s", key, depName, version)
+func getXrayLicenseSarifHeadline(depName, version, key, watch string) string {
+	headline := fmt.Sprintf("[%s] in %s %s", key, depName, version)
+	if watch != "" {
+		headline = fmt.Sprintf("%s (%s)", headline, watch)
+	} else {
+		headline = fmt.Sprintf("License violation %s", headline)
```

Comment:
is we don't have a watch we don't have a violation
mayne change the prefix to License Issue ?